src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -100,8 +101,8 @@
     public static final EditCommand.EditGuestDescriptor DESC_AMY;
     public static final EditCommand.EditGuestDescriptor DESC_BOB;
 
-    public static final String VALID_FLAG_ROOM = PREFIX_ROOM.toString();
-    public static final String VALID_FLAG_GUEST = PREFIX_GUEST.toString();
+    public static final String VALID_FLAG_ROOM = FLAG_ROOM.toString();
```

Comment:
What are the uses of `VALID_FLAG_ROOM` and `VALID_FLAG_GUEST`, vis-a-vis using the `FLAG_ROOM` and `FLAG_GUEST` as it is?

Reply:
For this you have to ask @JiaqingTan